h2/src/main/org/h2/command/Parser.java

```diff
@@ -4356,20 +4340,20 @@ private String readCommentIf() {
     }
 
     private Column parseColumnWithType(String columnName) {
-        String original = currentToken;
+        StringBuilder original = new StringBuilder(currentToken);
```

Comment:
It looks like this optimization may have nagetive overall effect, because in the most cases original string is not concatenated with different extensions.